README.md

```diff
@@ -73,217 +88,185 @@ With the virtual environment still activated, run the Django migrations, to set
     SECRET_KEY=""
-    
-Run migrations on Heroku like this:
 
-    heroku run "script/migrate" --app [heroku app name]
-    
 
-## Adding example data to database
+The Heroku-GitHub integration does not work with `git-submodules`
 
-There is included dummy data for demonstrating the app. 
-Run this command to insert it to your database.
+So to deploy we need to push to the Heroku `bluetail` app git remote manually.
 
-    script/insert_example_data
+This is easiest done using the Heroku CLI tools. https://devcenter.heroku.com/articles/heroku-cli
 
-or on heroku
-    
-    heroku run "script/insert_example_data" --app [heroku app name]
+1. Log in to Heroku CLI (https://devcenter.heroku.com/articles/heroku-cli#getting-started)
+2. Add the Heroku app git remote to your git clone
 
-    
-## Viewing example data
+    Execute this command in your bluetail clone root directory
 
-There are basic endpoints for viewing the sample data by ID
+        heroku git:remote --app bluetail
 
-OCDS release
+3. Force push your branch to the Heroku remote `master` branch.
 
-http://127.0.0.1:8000/ocds/ocds-123abc-PROC-20-0001/
+        git push heroku master:master --force
 
-BODS statements
+    Note you can push any local branch, but it must be pushed to the Heroku remote `master` branch to deploy.
 
-http://localhost:8000/bods/statement/person/17bfeb0d-4a63-41d3-814d-b8a54c81a1i/
-http://localhost:8000/bods/statement/entity/1dc0e987-5c57-4a1c-b3ad-61353b66a9b1/
-http://localhost:8000/bods/statement/ownership/676ce2ec-e244-409e-85f9-9823e88bc003/
+        git push heroku [local_branch_to_push]:master --force
 
+    If there are issues/errors from the Heroku git repo it can be reset first using https://github.com/heroku/heroku-repo
 
-### Filtering the dataset
+        heroku plugins:install heroku-repo
+        heroku repo:reset -a bluetail
 
-Each dataset has a distinct OCID prefix, as with publishers of real data.
+4. (Optional) Run the setup script to reset the Heroku database.
 
-https://standard.open-contracting.org/latest/en/schema/identifiers/#contracting-process-identifier-ocid
+        heroku run "script/setup"
 
-We can filter our tender listing to a specific dataset by specifying a prefix in the HTML parameter `ocid_prefix`:
+## Testing
 
-For example:
+There are just a few tests written to aid development.
+To run use this command
 
-    http://127.0.0.1:8000/ocds/?ocid_prefix=ocds-b5fd17
-    
-#### Prefixes used in the sample data:
+    script/manage test
 
-Prototype data:
+## Running
 
-    http://127.0.0.1:8000/ocds/?ocid_prefix=ocds-123abc-
+### Assets
 
-Contracts Finder data linked to Companies House where suppliers have a matching BODS record:
+Django‘s [staticfiles](https://docs.djangoproject.com/en/2.2/ref/contrib/staticfiles/) app copies all the static files (including sass files) to /static, and then [django-pipeline](https://django-pipeline.readthedocs.io/en/latest/index.html) compiles the sass files to css, in place, in /static
```

Comment:
For consistency, there should probably be backticks around `/static` here, both times it appears on this line.